torch/lib/THD/master_worker/master/Master.cpp

```diff
@@ -4,29 +4,33 @@
 #include "../worker/Worker.h"
 #include "../../process_group/General.hpp"
 
-
 namespace thd {
-
 namespace master {
 
 std::unique_ptr<MasterCommandChannel> masterCommandChannel;
 
 } // namespace master
-
 } // namespace thd
 
 using namespace thd;
+using namespace thd::master;
 
 bool THDMasterWorkerInit(THDChannelType channel_type) {
   if (!THDProcessGroupInit(channel_type)) return false;
 
   if (dataChannel->getRank() > 0) {
+    /*
+     * Worker initialization. If initialization succeeds it goes into
```

Comment:
Either say that it throws an exception otherwise, or just that it goes into an infinite loop.